File: dataset_reduzido/codenet/Java/Code/s193347311.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int intLoop = scan.nextInt();
		String strTaro = "";
		String strHana = "";
		int intTaro = 0;
		int intHana = 0;

		for(int i = 0; i < intLoop; i++){
			strTaro = scan.next();
			strHana = scan.next();
			int intJudge = strTaro.compareTo(strHana);
			if(intJudge < 0){
				intHana += 3;
			}else if(intJudge == 0){
				intTaro++;
				intHana++;
			}else if(0 < intJudge){
				intTaro += 3;
			}
		}
		System.out.println(intTaro + " " + intHana);
		scan.close();
	}
}

File: output/Qwen/codenet/Java/Python/s193347311.py
def main():
    int_loop = int(input())
    int_taro = 0
    int_hana = 0

    for _ in range(int_loop):
        str_taro = input()
        str_hana = input()
        int_judge = str_taro.compare_to(str_hana)
        if int_judge < 0:
            int_hana += 3
        elif int_judge == 0:
            int_taro += 1
            int_hana += 1
        elif 0 < int_judge:
            int_taro += 3

    print(int_taro, int_hana)

if __name__ == "__main__":
    main()